Cargo.toml

```
@@ -8,8 +8,8 @@ categories   = ["embedded", "no-std"]
 
 [dependencies]
 critical-section = "1.1.2"
-esp-hal = { version = "0.16.0" }
-heapless         = { version = "0.8.0",  default-features = false }
+esp-hal = { git = "https://github.com/esp-rs/esp-hal", rev = "8f2ee88" }
```

To be updated when `esp-hal` is released:
```suggestion
esp-hal          = "0.17.0"
```